apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/RedisClientConstructorInterceptor.java

```diff
@@ -30,6 +33,14 @@ public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
         RedisURI redisURI = (RedisURI) allArguments[1];
         RedisClient redisClient = (RedisClient) objInst;
         EnhancedInstance optionsInst = (EnhancedInstance) redisClient.getOptions();
-        optionsInst.setSkyWalkingDynamicField(redisURI.getHost() + ":" + redisURI.getPort());
+        StringBuilder redisInfoSb = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder redisPeer = new StringBuilder();
```
You should make the variable more clear.

Reply:
done